pvr.nextpvr/resources/language/resource.language.en_gb/strings.po

```diff
@@ -341,3 +341,7 @@ msgstr ""
 msgctxt "#30700"
 msgid "When disabled resume location and watched status will be managed only in Kodi"
 msgstr ""
+
+msgctxt "#30201"
+msgid "End time invalid.  Choose a recording duration"
```

Comment:
One space between the sentences.

Reply:
I'm an old guy so learned to type on a typewriter and this is automatic.  Fixed.